### PR TITLE
feat(gatsby-plugin-image): Add getLowResolutionImageURL plugin helper

### PR DESCRIPTION
**Pull request by ascorbic**
Adds a helper function to the image plugin toolkit, to allow plugin authors to get a URL for a low res image that matches the aspect ratio and crop of the main image. This is to allow them to download it as a blurred placeholder.

### CODE REVIEW

File: packages/gatsby-plugin-image/src/image-utils.ts [hunk 1]
```diff
@@ -194,11 +194,33 @@ export function setDefaultDimensions(
...
+  args: IGatsbyImageHelperArgs,
+  width = 20
+): string {
+  args = setDefaultDimensions(args)
+  const { generateImageSource, filename, aspectRatio } = args
+  return generateImageSource(
+    filename,
+    width,
+    Math.round(width / aspectRatio),
+    `jpg`,
```
Comment (laurieontech) [inline packages/gatsby-plugin-image/src/image-utils.ts:218]: Will this always be jpg?

File: packages/gatsby-plugin-image/src/image-utils.ts [hunk 2]
```diff
@@ -194,11 +194,33 @@ export function setDefaultDimensions(
...
   return { ...args, width, height, aspectRatio, layout, formats }
 }
 
+/**
+ * Use this for getting an image for the blurred placeholder. This ensures the
+ * aspect ratio and crop match the main image
+ */
+export function getLowResolutionImageURL(
+  args: IGatsbyImageHelperArgs,
+  width = 20
```
Comment (laurieontech) [inline packages/gatsby-plugin-image/src/image-utils.ts:210]: Any reason not to add this to the setDefaultDimensions call?

**Review by laurieontech — COMMENTED** [review]
Couple comments. Mostly ready to go though.

File: packages/gatsby-plugin-image/src/image-utils.ts (hunk 2 above)
Comment (ascorbic) [reply packages/gatsby-plugin-image/src/image-utils.ts:210]: We don't normally want to default to 20px images, just in this low res helper

Comment (laurieontech) [reply packages/gatsby-plugin-image/src/image-utils.ts:210]: Ah ok, realize that's reused now.

**Review by laurieontech — APPROVED** [review]

**Pull request merged**